assets/js/stripe-admin.js

```diff
@@ -86,6 +86,28 @@ jQuery( function( $ ) {
 					} );
 				} );
 			} );
+
+			// Add secret visibility toggles.
+			$( '#woocommerce_stripe_test_secret_key, #woocommerce_stripe_secret_key, #woocommerce_stripe_test_webhook_secret, #woocommerce_stripe_webhook_secret' ).after(
+				'<button class="wc-stripe-toggle-secret" style="height: 30px; margin-left: 2px"><span class="dashicons dashicons-visibility"></span></button>'
```

Comment:
What do you think of adding a `cursor: pointer` here to indicate that it can be clicked?